test/user/signup_signin.js

```diff
@@ -81,6 +79,40 @@ describe('POST api/auth/signup', () => {
   });
 });
 
+describe('POST /api/search/users', () => {
```

Comment:
I don't think this is  valid location for your tests. This file is supposed to be for sign up/sign in tests

Reply:
No remarks to this?

Reply:
The reason why i did that was because I needed some variables that were already present in that file.